step2/codes/src/component/templates/OrderTemplate.jsx

```diff
@@ -0,0 +1,185 @@
+                        </div>        
+                    </div>
+                </div>
+                <div className="border">
+                    <div className=" p-4 border-b">
+                        <span className="font-semibold text-lg">주문상품 정보 </span>
+                        <span>(총 {getTotalQuantity()}개)</span>
+                    </div>
+                    {/* 각 주문의 정보 */}
+                    <OrderItems />
+                    <div className="h-[46px] text-center leading-[46px]">
+                        <span className="text-sky-600">무료배송</span>
+                    </div>
+                </div>
+                {/* 총 주문 금액 */}
+                <div className="p-4 flex items-center justify-between border my-4">
+                    <h3 className="text-lg">총 주문 금액</h3>
+                    <span className="text-lg font-bold">{comma(totalPrice)}원</span>
+                </div>
+                {/* 전체 동의, 구매 조건 확인 및 결제 진행 동의 */}
+                <div className="flex flex-col gap-2 border">
+                    <div className="flex gap-2 border-b p-4 align-center">
+                        <input type="checkbox" id = "all-agree" 
+                            ref={allAgreeRef}
+                            checked={agreePayment && agreePolicy}
+                            onClick = {handleAllAgree}
+                            onChange={()=> {}}
+                            className="w-[22px] h-[22px] mt-[7px]"
+                        />
+                        <label htmlFor="all-agree" className="text-xl font-bold">전체 동의</label>
+                    </div>
+                    <div className="flex gap-2 px-4">
+                        <input type="checkbox" id="agree" name = "payment-agree"
+                            ref={agreePaymentRef}
+                            checked={agreePayment}
+                            onChange={handleAgreement}
+                            className="w-[22px] h-[22px] mt-[4px] border-[0.5px] border-red-600"
+                        />
+                        <label htmlFor="agree" className="text-sm ">구매 조건 확인및 결제 진행 동의</label>
+                    </div>
+                    <div className="flex gap-2 px-4">
+                        <input type="checkbox" id="policy" name="policy-agree"
+                            ref={agreePolicyRef}
+                            checked={agreePolicy}
+                            onChange={handleAgreement}
+                            className="w-[22px] h-[22px] mt-[4px] border-[0.5px] "
+                        />
+                        <label htmlFor="policy" className="text-sm">개인정보 제 3자 제공 동의</label>
+                    </div>
+                    {/* 결제하기 버튼 */}
+                    <button 
+                        onClick={() => {
+                            // 동의가 이루어지지 않은 경우 처리
+                            if (agreePayment === false || agreePolicy === false) {
```

Comment:
여기도 
```suggestion
if (!agreePayment || !agreePolicy) {
```
이렇게 간소화 해주시면 더 간결해지겠습니다~!